data/json/recipes/armor/legs.json

```
@@ -655,5 +655,44 @@
     "book_learn": [ [ "manual_tailor", 1 ] ],
     "using": [ [ "sewing_standard", 8 ] ],
     "components": [ [ [ "rag", 4 ] ] ]
+  },
+  {
+    "result": "skirt_armor",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_LEGS",
+    "skill_used": "tailor",
+    "difficulty": 4,
+    "time": "30 m",
+    "autolearn": true,
+    "book_learn": [ [ "textbook_armwest", 3 ] ],
+    "using": [ [ "sewing_standard", 15 ] ],
+    "components": [ [ [ "rag", 10 ] ] ]
+  },
+  {
+    "result": "skirt_armor_leather",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_LEGS",
+    "skill_used": "tailor",
+    "difficulty": 4,
+    "time": "30 m",
+    "autolearn": true,
+    "book_learn": [ [ "textbook_armwest", 3 ] ],
+    "using": [ [ "sewing_standard", 15 ] ],
+    "components": [ [ [ "leather", 10 ] ] ]
+  },
+  {
+    "result": "skirt_armor_plate",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_LEGS",
+    "skill_used": "tailor",
+    "difficulty": 4,
+    "time": "15 m",
+    "autolearn": true,
+    "book_learn": [ [ "textbook_armwest", 3 ] ],
+    "using": [ [ "sewing_standard", 10 ], [ "steel_tiny", 2 ] ],
+    "components": [ [ [ "skirt_armor", 1 ] ] ]
```

```suggestion
    "components": [ [ [ "skirt_armor_leather", 1 ] ] ]
```
Item name and description implies it's meant to use the leather version as a base?